server/src/routes/api/v1/reviewsRouter.js

```diff
@@ -0,0 +1,15 @@
+import express from "express"
+import Review from "../../../models/Review.js"
+
+const reviewsRouter = new express.Router()
+
+reviewsRouter.delete("/:id", async (req, res) => {
+  try {
+    await Review.query().deleteById(req.params.id)
```

Comment:
We need to also make sure on the back end that the user who created the review is the one with access to delete it - so checking `req.user` compared to the Reviews `userId` before actually deleting